wowchemy/archetypes/slides/index.md

```diff
@@ -12,7 +12,42 @@ slides:
   theme: black
   # Choose a code highlighting style (if highlighting enabled in `params.toml`)
   #   Light style: github. Dark style: dracula (default).
-  highlight_style: dracula
+  #   Available highlight themes listed in: https://highlightjs.org/static/demo/
+  #   Use lower case names and replace space with hyphen '-'
+  highlight_style: monokai
```

Comment:
What's the reason for changing the default highlight theme?

Reply:
the comments are there to direct user to the list of available highlights.

I copied what I had set while testing things in the test slide.

I can set back to default